handlers.go

```diff
@@ -869,6 +869,17 @@ func (s *Server) handleAdd(c echo.Context, u *util.User) error {
 		return err
 	}
 
+	// if upload size is smaller than (min deal size / staging bucket file count limit)
```

Comment:
I made a suggestion on how to deal with this [here](https://github.com/application-research/estuary/pull/535#issuecomment-1318197048), I think it is better than rejecting uploads